modules/QtQuick/Controls/Styles/Material/MaterialEditMenu.qml

```diff
@@ -0,0 +1,178 @@
+/*
+ *   Copyright (C) 2015 by Aleix Pol Gonzalez <aleixpol@kde.org>
+ *   Copyright (C) 2015 by Marco Martin <mart@kde.org>
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU Library General Public License as
+ *   published by the Free Software Foundation; either version 2, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU Library General Public License for more details
+ *
+ *   You should have received a copy of the GNU Library General Public
+ *   License along with this program; if not, write to the
+ *   Free Software Foundation, Inc.,
+ *   51 Franklin Street, Fifth Floor, Boston, MA  2.010-1301, USA.
+ */
+
+import QtQuick 2.1
+import QtQuick.Controls 1.1
+import QtQuick.Controls.Styles 1.1
+import QtQuick.Window 2.1
+import QtQuick.Layouts 1.0
+import Material 0.1
+
+Item {
+    id: root
+    anchors.fill: parent
+
+    property bool editing: true
+    onEditingChanged: {
```

Comment:
The [QML coding conventions](http://doc.qt.io/qt-5/qml-codingconventions.html) doesn't specify the order of signal handlers, but I like to have signal listeners grouped together above functions and below properties.